pkg/apis/core/types_shoot.go

```diff
@@ -523,6 +523,34 @@ type KubeAPIServerConfig struct {
 	EnableAnonymousAuthentication *bool
 	// EventTTL controls the amount of time to retain events.
 	EventTTL *metav1.Duration
+	// AccessControl provides authorization mechanisms for Shoot-API-Server.
```

Comment:
Even if we are "mimicing" Istio here, it's actually not really about authorization, right? Let's be more clear so that end-users can easier understand:

```suggestion
	// AccessControl provides access control configurations for the API server.
```